src/pybind/mgr/dashboard/README.rst

```diff
@@ -34,5 +34,7 @@ Working on the Dashboard Code
 -----------------------------
 
 If you're interested in helping with the development of the dashboard, please
-see /doc/dev/dev_guide/dash_devel.rst for details on how to set up a development
-environment and other development-related topics.
+see ``/doc/dev/dev_guide/dash_devel.rst`` or the `online version
```

Comment:
```suggestion
see [``/doc/dev/dev_guide/dash_devel.rst``](https://ceph.readthedocs.io/en/latest/dev/developer_guide/dash-devel/)
```

Reply:
We need to all get on the same page with respect to whether we're going to use https/ceph.readthedocs.io or https://docs.ceph.com.

My understanding of the situation is that the readthedocs site doesn't have the API documentation that the ceph.docs.com site does. So anyone who wants API documentation is going to have to go to docs.ceph.com. Of course, docs.ceph.com has a really crummy search feature which only finds the first word in a multiword string, so if you look up (for instance) "ceph ls", you get the results for "ceph". And that's obviously worse than having no search feature at all, because it raises expectations before dashing them.

This comment won't solve this issue, but I want to start making more people aware of it so that we can find a solution.

Reply:
[OT] if we are not going to use RTD, i will stop maintaining the build.

Reply:
I think we came to the conclusion that ceph.readthedocs.io was supposed to become the canonical documentation source, while we keep docs.ceph.com for the generated content that can't be migrated to RTD.